### ISSUE TITLE
I seem to be getting constant re-loads on Android

### ISSUE THREAD

**Issue by adamgins** [percolatestudio/safe-reload#1]
Thanks for this package. It looks like this is the stuff that would be part of Meteor 1.2 core. looking forward to that.

Anyway, I am having some issues debugging where the issue is, so apologies about the vagueness of this request. 

On iOS my app seems to work fine. On Android, it's another matter, something seems to getting it to reload all the time. I need some help, please, to debug this.  

As a side issue, I can't seem to get chrome://remote to work to try and debug this and get more valid info for this request:
![image](https://cloud.githubusercontent.com/assets/7233741/8898282/0cbb92c0-3465-11e5-91e3-e00c4057b7af.png)

Any help appreciated

**Comment by dnish** [comment]
Yep, the same here, plugin is not working on Android with Cordova - after a HCP, it reloads the whole time.

**Comment by jaskinn** [comment]
I experience this situation on both iOS and Android

**Comment by udiedrichsen** [comment]
I got also with warning after add this packages to my meteor project:
cp: dest file already exists: .meteor/local/cordova-build/plugins/com.meteor.cordova-update/plugin.xml

skipping existing file: Asset.java
skipping existing file: AssetUriRemapper.java
skipping existing file: CordovaUpdatePlugin.java
skipping existing file: FilesystemUriRemapper.java
skipping existing file: UriRemapper.java
skipping existing file: Utils.java
skipping existing file: CordovaUpdate.m
skipping existing file: METEORCordovaURLProtocol.h
skipping existing file: METEORCordovaURLProtocol.m
skipping existing file: index.html
skipping existing file: index.html
skipping existing file: CordovaUpdate.js
=> Started your app.                                                            

=> App running at: http://localhost:3000/

**Comment by timbotnik** [comment]
Just published an updated version, and should fix these issues.  Let me know if others are still having problems.  New version is compatible with Meteor 1.2.

**Comment by dnish** [comment]
@timbotnik Yeah, the reload problem was fixed :). But I've got a broken app version on my device (while using the dev machine). I only see a white screen, before the app didn't show any image assets. After restarting it, I only get a blank page. Seems like the plugin doesn't fix all broken HCPs?

**Comment by timbotnik** [comment]
@dnish - thanks for the update.  Yes there are definitely certain types of things that can slip past `safe-reload`, such as routing errors, broken templates, etc.  In those cases at least autoupdate should be functioning (Hot Code Reload) so if you deploy a new version it should be able to recover.  Perhaps if you have an example of a case where it is not able to perform Hot Code Reload, that would be interesting to look at.

**Comment by dnish** [comment]
@timbotnik Mhhh can't reproduce it exactly, it happens just sometimes. Just got the following message in my console:

20151119-13:10:57.675(1) (android:http://meteor.local/packages/ddp-client.js:1181) Uncaught Error: INVALID_STATE_ERR

If I remember correct, I've killed the app while I did a change on my local dev machine. Maybe the interruption of this update causes the error?

//Edit
It happened again, here another output:

```
=> Client modified -- refreshing (x7)I20151119-14:14:56.553(1) (android:http://meteor.local/packages/meteor.js:906) Clock discrepancy detected. Attempting re-sync.
I20151119-14:15:01.016(1) (android:http://meteor.local/app/client/templates/dashboard/pages/followers/template.followers.js:1) Uncaught SyntaxError: Unexpected token <
I20151119-14:15:01.261(1) (android:http://meteor.local/app/client/templates/dashboard/pages/followers/followers.js:1) Uncaught SyntaxError: Unexpected token <
I20151119-14:15:04.980(1) (android:http://meteor.local/packages/blaze.js:3155) Uncaught Error: No such template: followers
I20151119-14:15:05.015(1) (android:http://meteor.local/packages/dispatch_kernel.js:336) Uncaught Error: Can't call View#autorun from a Tracker Computation; try calling it from the created or rendered callback
I20151119-14:15:05.022(1) (android:http://meteor.local/packages/dispatch_kernel.js:336) Uncaught Error: Can't call View#autorun from a Tracker Computation; try calling it from the created or rendered callback
I20151119-14:15:05.028(1) (android:http://meteor.local/packages/dispatch_kernel.js:336) Uncaught Error: Can't call View#autorun from a Tracker Computation; try calling it from the created or rendered callback

```
